<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCallShvMethod</class>
 <widget class="QDialog" name="DlgCallShvMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call SHV method</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Shv path</string>
       </property>
       <property name="buddy">
        <cstring>edShvPath</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Method</string>
       </property>
       <property name="buddy">
        <cstring>edMethod</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Parameters</string>
       </property>
       <property name="buddy">
        <cstring>edParams</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btCall">
       <property name="text">
        <string>&amp;Call</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="edShvPath">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="edMethod">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="edParams">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Response</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtResponse"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edShvPath</tabstop>
  <tabstop>edMethod</tabstop>
  <tabstop>edParams</tabstop>
  <tabstop>btCall</tabstop>
  <tabstop>txtResponse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
